tests/example-empty-file/expected_results.json

```diff
@@ -1 +1 @@
-{"message":"[error] /solution/src/test/scala/ExampleEmptyFileTest.scala:7:5: not found: value Leap\n[error]     Leap.leapYear(2015) should be (false)\n[error]     ^\n[error] /solution/src/test/scala/ExampleEmptyFileTest.scala:12:5: not found: value Leap\n[error]     Leap.leapYear(1996) should be (true)\n[error]     ^\n[error] /solution/src/test/scala/ExampleEmptyFileTest.scala:17:5: not found: value Leap\n[error]     Leap.leapYear(2100) should be (false)\n[error]     ^\n[error] /solution/src/test/scala/ExampleEmptyFileTest.scala:22:5: not found: value Leap\n[error]     Leap.leapYear(2000) should be (true)\n[error]     ^\n[error] four errors found\n[error] (Test / compileIncremental) Compilation failed\n[error] \n","version":2,"status":"error"}
\ No newline at end of file
+{"message":"/tmp/exercise/src/test/scala/ExampleEmptyFileTest.scala:7: error: not found: value Leap\n    Leap.leapYear(2015) should be (false)\n    ^\n/tmp/exercise/src/test/scala/ExampleEmptyFileTest.scala:12: error: not found: value Leap\n    Leap.leapYear(1996) should be (true)\n    ^\n/tmp/exercise/src/test/scala/ExampleEmptyFileTest.scala:17: error: not found: value Leap\n    Leap.leapYear(2100) should be (false)\n    ^\n/tmp/exercise/src/test/scala/ExampleEmptyFileTest.scala:22: error: not found: value Leap\n    Leap.leapYear(2000) should be (true)\n    ^\n4 errors\n","version":2,"status":"error"}
```

Comment:
Maybe there could be a post-processing step to change `/tmp/exercise` to `/solution`?

Reply:
Being a bit lazy, would directly creating a `/solution` folder and working from there inside the docker image also acceptable? :smile_cat:

Reply:
No, sorry, as the `/solution` folder is automatically being mapped.

Reply:
Ah, I missed that. Will add the post-processing.